### ISSUE TITLE
Ctrl F should focus the search field

### ISSUE THREAD

**Issue by koppor** [PhilippC/keepass2android#844]
OS: Chrome OS (refs https://github.com/PhilippC/keepass2android/issues/272)

When I press <kbd>Ctrl</kbd> + <kbd>F</kbd>, the usual behavior on Windows is that the search field is focused. On CHromeOS, the app selects nothing.

Desired behavior:

User presses <kbd>Ctrl</kbd> + <kbd>F</kbd>.

If search field is focused {
  select all text
} else {
  focus search field
}

**Comment by ChrisPurves** [comment]
I second this.  I'm using keepass2android on a chromebook and it is very annoying to have to click on the search icon.  I would much prefer a keyboard shortcut. How about CTRL-F?

**Comment by koppor** [comment]
Yeah, <kbd>Ctrl</kbd>+<kbd>F</kbd> is also my desired keyboard shortcut (see "desired behavior" in my report).

**Comment by twopoint71** [comment]
+1.  I'll buy you another beer. <3
